modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/numeric/NumericDDMFormFieldTemplateContextContributorTest.java

```diff
@@ -242,6 +242,21 @@ public void testGetNumericInputMaskProperties() {
 			String.valueOf(parameters.get("symbols")));
 	}
 
+	@Test
+	public void testGetPredefinedValue() {
+		DDMFormField ddmFormField = new DDMFormField("field", "numeric");
+
+		ddmFormField.setProperty(
+			"predefinedValue",
+			DDMFormValuesTestUtil.createLocalizedValue("42", _locale));
+
+		Assert.assertEquals(
+			DDMFormValuesTestUtil.createLocalizedValue("42", _locale),
+			_numericDDMFormFieldTemplateContextContributor.getPropertyValue(
+				ddmFormField, new DDMFormFieldRenderingContext(),
+				"predefinedValue"));
```

Comment:
In addition, instead of using the `getPropertyValue` function, you should do:

```
DDMFormField ddmFormField = new DDMFormField("field", "numeric");

ddmFormField.setProperty("predefinedValue", DDMFormValuesTestUtil.createLocalizedValue("42", _locale));

DDMFormFieldRenderingContext ddmFormFieldRenderingContext = new DDMFormFieldRenderingContext();

ddmFormFieldRenderingContext.setLocale(_locale);
ddmFormFieldRenderingContext.setViewMode(true);

Map<String, Object> parameters =
	_numericDDMFormFieldTemplateContextContributor.getParameters(ddmFormField, ddmFormFieldRenderingContext);
		
Assert.assertEquals("42", String.valueOf(parameters.get("predefinedValue")));
```